src/Command/SendChangeAlertsCommand.php

```diff
@@ -224,15 +241,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
      */
     protected function getTemplatePath(SchoolInterface $school)
     {
-        $paths = [
```

Comment:
this is an incomplete conversion, the ability to override the instance-wide template (not the school-prefixed one) is lost.

Reply:
Because the `%kernel.project_dir%/custom/templates` directory is registered before the symfony standard `templates` one it will look first for exact match templates in that directory. So system wide overrides will work without us needing to deal with them. We only have to account for the school specific special paths.

Reply:
thanks for the explanation.